docs/integration_resources/resources/usd_tutorials/usd_tutorial_04.md

```diff
@@ -0,0 +1,82 @@
+
+<figure>
+    <img src="images/usd_tutorial_04/step_03_t04.png" alt="Added 'SaveAsUsdaFile' node to the board connected to 'OpenUsdFile' node.">
+    <figcaption><b>Figure 3</b>: Added 'SaveAsUsdaFile' node to the board connected to 'OpenUsdFile' node.</figcaption>
+</figure>
+
+4. Execute the workflow. Doing this, a new file with the `.usda` extension will be generated using as information the `.usd` file. The board should look like the following:
+
+<figure>
+    <img src="images/usd_tutorial_04/step_04_t04.png" alt="Executed the workflow for creating the '.usda' file.">
+    <figcaption><b>Figure 4</b>: Executed the workflow for creating the '.usda' file.</figcaption>
+</figure>
+
+5. Add another new *UsdStageOpen* node to the board and rename it as *OpenUsdaFile*. Set the value for the **filepath** input plug of the current node to point to the file generated by the *SaveAsUsdaFile* node, meaning having in the current example the value: `D:/USD_TUTORIALS/04/Sphere.usda`. The board should look like the following:
+
+<figure>
+    <img src="images/usd_tutorial_04/step_05_t04.png" alt="Added 'OpenUsdaFile' node to the board.">
+    <figcaption><b>Figure 5</b>: Added 'OpenUsdaFile' node to the board.</figcaption>
+</figure>
+
+6. To save a `.usdc` file, add a new *UsdStageSave* node to the board and rename it as *SaveAsUsdcFile*. Connect the **stage** output plug from the *OpenUsdaFile* node to the **stage** input plug of the new node. Set as the value for the **filepath** input plug a valid filepath to a `.usdc` file, such as `D:/USD_TUTORIALS/04/Sphere.usdc`. The board should look like the following:
+
+<figure>
+    <img src="images/usd_tutorial_04/step_06_t04.png" alt="Added 'SaveAsUsdcFile' node to the board connected to 'OpenUsdaFile' node.">
+    <figcaption><b>Figure 6</b>: Added 'SaveAsUsdcFile' node to the board connected to 'OpenUsdaFile' node.</figcaption>
+</figure>
+
+7. Execute the workflow again. Now, the workflow will generate a new file with the `.usdc` extension will be generated using the information from the `.usda` file. The board should look like the following:
+
+<figure>
+    <img src="images/usd_tutorial_04/step_07_t04.png" alt="Executed the workflow for creating the '.usdc' file.">
+    <figcaption><b>Figure 7</b>: Executed the workflow for creating the '.usdc' file.</figcaption>
+</figure>
+
+
+### Run verifications
+
+1. For running the verification between the original `Sphere.usd` file and the generated file `Sphere.usda`, use the terminal for executing the *usddiff* tool shipped with USD for showing the difference between 2 USD files. The terminal should show no differences:
```

Comment:
This will be a problem for the average user as the usd toolset shipped with shift is not sourced in any terminal. Maybe we should have a shift_console.bat ??? @Inbibo/shift